server/api/sendGrid.js

```diff
@@ -5,15 +5,21 @@ const config = require("../config/config.js")
 
 sgMail.setApiKey(config.SENDGRID_API_KEY)
 
-function sendAnEmail(recipient, sender, subject, text) {
+function sendAnEmail(req, res) {
     const msg = {
-        to: recipient,
-        from: sender,
-        subject: subject,
-        text: text,
-        html: `<strong>${text}</strong>`,
+        to: req.body.recipient,
+        from: req.body.sender,
+        subject: req.body.subject,
+        text: req.body.text,
+        html: `<strong>${req.body.text}</strong>`,
     }
-    sgMail.send(msg)
+    sgMail
+        .send(msg)
+        .then(() => {
+            console.log("mail sent successfully")
+            res.end()
```

Comment:
- [ ] can we send a message to the client side telling them that the email has been sent with the correct status for it?

Reply:
@JawharBairakdar yeah you can do that but not from here, you will need to modify the new function  sendAnEmail() in src/util/index.js (front-end), depends on what we need like set a state for a popup window when the email is sent

Reply:
Actully all of what we need is only the state and a message that says the email has been sent.
you can do it from a multible places it's your code you can chose what you prefer 🙂.

I am Actully wondering if you can send a status code if the email has been faild for a reason so the sender can know that it didn't arrive.